lib/jwt/jwk/rsa.rb

```diff
@@ -51,6 +51,10 @@ def verify_key
       def export(options = {})
         exported = parameters.clone
         exported.reject! { |k, _| RSA_PRIVATE_KEY_ELEMENTS.include? k } unless private? && options[:include_private] == true
+
+        exported[:x5c] = Base64.strict_encode(rsa_key.to_der) if options[:x5c]
```

Comment:
About the naming of the options, would it make sense to have the prefix `include_` on the options? To be inline with the `include_private`

Reply:
That makes sense, let me update it.